.Jenkins/workflows/Jenkinsfile_EL9

```diff
@@ -227,7 +227,7 @@ pipeline {
                     }
                     post {
                         always {
-                            archiveArtifacts artifacts: "dist/decisionengine*,rpmbuild/RPMS/decisionengine*"
+                            archiveArtifacts artifacts: "dist/hepcloud_de*,rpmbuild/RPMS/decisionengine*"
```

Comment:
I would use dist/hepcloud* because it is hepcloud-de... for wheels and hepcloud_de... for the tar

Reply:
In the output folder I see both wheels and tarball have `hepcloud_de` in their filenames.
But in case it would change, I followed your suggestion and updated the code by useing the `dist/hepcloud*` pattern.